### DISCUSSION TITLE
Add new boards - Franzininho WiFi

### DISCUSSION THREAD

**Discussion by FBSeletronica** [mu-editor/mu#1394] [Ideas & New Features]
Hello,

recently I added 2 new boards on CircuityPython. I would like to add them on Mu. The boards are:
https://circuitpython.org/board/franzininho_wifi_wroom/
https://circuitpython.org/board/franzininho_wifi_wrover/

Can I do it? 

Thanks

**Comment by ajsirota** [comment]
**I think** you can add the VID/PIDs to the array and submit a PR.
https://github.com/mu-editor/mu/blob/master/mu/modes/circuitpython.py#L43-L74

**Reply by FBSeletronica** [reply]
Nice! I will dot it. Thank you!
